README.md

```diff
@@ -155,11 +193,60 @@ Returns an array of results - one item for each item in `patches`. The type of e
 * `remove`, `replace` and `move` - original object that has been removed
 * `add` (only when adding to an array) - index at which item has been inserted (useful when using `-` alias)
 
-#### jsonpatch.observe (`obj` Object, `callback` Function (optional)) : `observer` Object
+## Object observing
+### ES6-Proxy version 
+#### constructor: JsonObserver( `root` Object ):  JsonObserver
+
+Available in *json-observe.js* 
+
+Creates an instance of `JsonObserver` around your object of interest `root`, for later `observe`, `unobserve`, `switchCallbackOff`, `switchCallbackOn` calls.
+Returns `JsonObserver`.
+
+#### JsonObserver.observe(`record` boolean, [`callback` Function]): Proxy
```

Comment:
`observe` is a method of an instance, not a static method of the JsonObserver, isn't it?
Then I'd rather use JSDoc convention: `JsonObserver#observe`